source/includes/records_errors.md

```diff
@@ -0,0 +1,75 @@
+# Record error
+
+## The record error object
+
+The record error describes errors which occured during the processing. An error comes from:
+
+* a processor: when a value is valid or an operation failed
+* a type conversion: when a value cannot be converted. For example the string "s" converted to an integer.
+
+
+### Attributes
+
+Attribute | Description
+--------- | -----------
+`record_id` <br> *string* | Id of the record after processing
+`processor_uid` <br> *string* | Uid of the processor who generates the error
+`field_uid` <br> *string* | Uid of the field involved in the error
+`raw_message` <br> *string* | Message with parameters template
+`raw_params` <br> *[params object](#params-object)* | Json object with parameters templates values
+`message` <br> *string* | Message with replaced parameters
+
+## Retrieve the records errors
+
+```HTTP
+GET https://{YOURDOMAIN}.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}/status/records_errors
+```
+
+> Example request
+
+```HTTP
+curl https://yourdomain.opendatasoft.com/api/management/v2/datasets/da_XXXXXX/records_errors \
+    -u username:password
+```
+
+> Example response
+
+```json
+{
+    "record_id" : "442cdf84ec84de2474dfba9422b045c064c36f05",
+    "processor_uid": "pr_XXXXXX",
+    "field_uid": "column_1",
+    "message": "Cannot convert \"a\" to \"int\"",
+    "raw_message": "Cannot convert \"{value}\" to \"{type}\"",
+    "raw_params": {
+        "value": "a",
+        "type": "int"
+    }
+}
+```
+
+Retrieves the dataset records errors generated by the last publish.
+
+### Parameters
+
+No parameters
+
+### Returns
+
+The list of records errors object that applies to the given dataset. See [records errrors object](#records_errors) for specific attributes and the list of objects.
```

Comment:
The list of record error objects that apply to the given dataset
In square brackets, there's a r too many to "errrors".